test/Pester/PlatyPs.Tests.ps1

```diff
@@ -379,6 +385,54 @@ Describe 'New-MarkdownHelp' {
 
         }
     }
+	
+	Context 'Full Type Name' {
+        function global:Get-Alpha
+        {
+            param(
+                [Switch]
+                [Parameter(Position=1)]
+                $WhatIf,
+                [string]
+                [Parameter(Position=2)]
+                $CCC
+            )
+        }
+
+        It 'use full type name when specified' {
+            $expectedParameters = normalizeEnds @'
+Type: System.String
+Type: System.Management.Automation.SwitchParameter
+
+'@ 
+            $expectedSyntax = normalizeEnds @'
+Get-Alpha [[-WhatIf] <System.Management.Automation.SwitchParameter>] [[-CCC] <System.String>]
```

Comment:
This `[[-WhatIf] <System.Management.Automation.SwitchParameter>]` doesn't sound like a good idea.
Is that's the behavior you want with `-UseFullTypeName`? cc @joeyaiello